types/src/main.ts

```diff
@@ -2611,6 +2611,92 @@ export interface TextDocumentContentChangeEvent {
 	text: string;
 }
 
+/**
+ * Denotes a location of an editor in the window. Editors can be arranged in a grid
+ * and each column represents one editor location in that grid by counting the editors
+ * in order of their appearance.
+ */
+export enum ViewColumn {
```

Comment:
This exposes implementation details of VS Code at the protocol level, which doesn't feel right. 

If you really need to control VS Code-specific editor placement, a better approach could be to make a custom command in your VS Code extension.

This probably won't even work for VS Code, given the recent switch to grid layout.

Reply:
I don't really need a specific editor placement, so I'm simply omitting it for the moment.